### ISSUE TITLE
Sonos plugin

### ISSUE THREAD

**Issue by gerard33** [tester22/Domoticz-Sonos#6]
@tester22, I saw you update the Readme that the project is no longer maintained.
I already did some updates in my fork of your repository so radio stations and volume are shown.
Shall I continue developing and maintaining the code for the Sonos plugin on my own Github?

**Comment by tester22** [comment]
Sounds good.

I have swithched to another automation software.

2017-07-18 0:07 GMT+02:00 gerard33 <notifications@github.com>:

**Comment by gerard33** [comment]
Allright, will post it on the Domoticz forum then as well.
Thanks for your work on this plugin!!!

Did you go to Home Assistant?

**Comment by tester22** [comment]
Yep, some work to set it up but so far everything I wanted to connect works
out of the box.


tis 18 juli 2017 kl. 00:29 skrev gerard33 <notifications@github.com>:

**Comment by gerard33** [comment]
New version of the plugin is available on my Github page https://github.com/gerard33/sonos.
